client/state/sites/selectors.js

```diff
@@ -939,6 +939,13 @@ export function siteHasMinimumJetpackVersion( state, siteId ) {
 	}
 
 	const siteJetpackVersion = getSiteOption( state, siteId, 'jetpack_version' );
+
+	// FIXME: Recently connected Jetpack sites have jetpack_version = false
+	// When this is corrected, drop this condition
```

Comment:
On second thought, this probably doesn't need to be removed ever. I think this `FIXME` can be removed.